[Bug]: Cannot execute Doris Bulkd Loader

### Apache Hop version?

2.2.0

### Java version?

➜  soft java -version openjdk version "11.0.16" 2022-07-19 OpenJDK Runtime Environment Temurin-11.0.16+8 (build 11.0.16+8) OpenJDK 64-Bit Server VM Temurin-11.0.16+8 (build 11.0.16+8, mixed mode)

### Operating system

macOS

### What happened?

When I try to execute the doris bulk loader, I get an error.

This is the process

<img width="976" alt="image" src="https://user-images.githubusercontent.com/2102238/210170509-8d5db2e7-1985-4acc-9540-f778a48e8c5b.png">

Then I run the pipeline,

get an error
2023/01/01 13:21:16 - Hop - Launching pipeline [loadtest]...
2023/01/01 13:21:16 - Hop - Started the pipeline execution.
2023/01/01 13:21:16 - loadtest - Executing this pipeline using the Local Pipeline Engine with run configuration 'local'
2023/01/01 13:21:16 - loadtest - Execution started for pipeline [loadtest]
2023/01/01 13:21:16 - Data grid.0 - Finished processing (I=0, O=0, R=0, W=3, U=0, E=0)
2023/01/01 13:21:16 - Doris bulk loader.0 - ERROR: Because of an error, this transform can't continuejava.lang.IllegalArgumentException
2023/01/01 13:21:16 - Doris bulk loader.0 - ERROR: org.apache.hop.pipeline.transforms.dorisbulkloader.DorisStreamLoadException: java.lang.IllegalArgumentException
2023/01/01 13:21:16 - Doris bulk loader.0 - 	at org.apache.hop.pipeline.transforms.dorisbulkloader.DorisBulkLoader.processStreamLoad(DorisBulkLoader.java:181)
2023/01/01 13:21:16 - Doris bulk loader.0 - 	at org.apache.hop.pipeline.transforms.dorisbulkloader.DorisBulkLoader.processRow(DorisBulkLoader.java:76)
2023/01/01 13:21:16 - Doris bulk loader.0 - 	at org.apache.hop.pipeline.transform.RunThread.run(RunThread.java:55)
2023/01/01 13:21:16 - Doris bulk loader.0 - 	at java.base/java.lang.Thread.run(Thread.java:829)
2023/01/01 13:21:16 - Doris bulk loader.0 - Caused by: java.lang.IllegalArgumentException
2023/01/01 13:21:16 - Doris bulk loader.0 - 	at java.base/java.util.concurrent.ArrayBlockingQueue.<init>(ArrayBlockingQueue.java:269)
2023/01/01 13:21:16 - Doris bulk loader.0 - 	at java.base/java.util.concurrent.ArrayBlockingQueue.<init>(ArrayBlockingQueue.java:254)
2023/01/01 13:21:16 - Doris bulk loader.0 - 	at org.apache.hop.pipeline.transforms.dorisbulkloader.RecordBuffer.<init>(RecordBuffer.java:41)
2023/01/01 13:21:16 - Doris bulk loader.0 - 	at org.apache.hop.pipeline.transforms.dorisbulkloader.RecordStream.<init>(RecordStream.java:37)
2023/01/01 13:21:16 - Doris bulk loader.0 - 	at org.apache.hop.pipeline.transforms.dorisbulkloader.DorisStreamLoad.<init>(DorisStreamLoad.java:91)
2023/01/01 13:21:16 - Doris bulk loader.0 - 	at org.apache.hop.pipeline.transforms.dorisbulkloader.DorisBulkLoader.initStreamLoad(DorisBulkLoader.java:221)
2023/01/01 13:21:16 - Doris bulk loader.0 - 	at org.apache.hop.pipeline.transforms.dorisbulkloader.DorisBulkLoader.processStreamLoad(DorisBulkLoader.java:151)
2023/01/01 13:21:16 - Doris bulk loader.0 - 	... 3 more
2023/01/01 13:21:16 - Doris bulk loader.0 - Finished processing (I=0, O=0, R=1, W=0, U=0, E=1)
2023/01/01 13:21:16 - loadtest - Pipeline duration : 0.088 seconds [  0.088" ]
2023/01/01 13:21:16 - loadtest - Execution finished on a local pipeline engine with run configuration 'local'
2023/01/01 13:21:16 - loadtest - Pipeline detected one or more transforms with errors.
2023/01/01 13:21:16 - loadtest - Pipeline is killing the other transforms!

And then, when I review the properties again

<img width="974" alt="image" src="https://user-images.githubusercontent.com/2102238/210170534-6dcb604a-d012-4f30-b938-8d64fc3edfe5.png">

The propertie datafield is empty.



### Issue Priority

Priority: 3

### Issue Component

Component: Actions

--- a/plugins/transforms/dorisbulkloader/src/main/java/org/apache/hop/pipeline/transforms/dorisbulkloader/DorisBulkLoaderMeta.java
+++ b/plugins/transforms/dorisbulkloader/src/main/java/org/apache/hop/pipeline/transforms/dorisbulkloader/DorisBulkLoaderMeta.java
@@ -83,7 +83,7 @@ public class DorisBulkLoaderMeta extends BaseTransformMeta<DorisBulkLoader, Dori
   @HopMetadataProperty private int bufferCount;
 
   /** doris stream load data fieldname */
-  private String dataField;
+  @HopMetadataProperty private String dataField;
 
   public DorisBulkLoaderMeta() {
     super(); // allocate BaseTransformMeta

